index.php

```diff
@@ -1,15 +1,3 @@
 <?php
-
-define('ROOT_DIR', realpath(dirname(__FILE__)) . '/');
```

Comment:
Well that is roughly what I had in mind when spoting these constants. Looks tidy now :)